doc/ext/docscrape_sphinx.py

```diff
@@ -129,6 +129,8 @@ def _str_member_list(self, name):
             #     out += [''] + autosum
 
             if others:
+                out += [r'.. tabularcolumns:: p{3cm}p{\dimexpr\linewidth-3cm-4\tabcolsep\relax}']
+                out += ['.. rst-class:: longtable']
```

Comment:
The tabularcolumns directive has impact only on latex builder. The added longtable class in HTML output does not harm.